### DISCUSSION TITLE
Changes in one view automatically apply to another view where I didn't change anything

### DISCUSSION THREAD

**Discussion by markvrc** [community/community#28753] [Projects and Issues] [answered]
Steps to reproduce:
1. Create a project, and a first view
2. Add some issues to the first view
3. Create a new view within the same project
4. Add some more issues to the second view.

Expected result:
- View one has only those issues that were added on step 2. 
- View two has the issues that were added on the step 2 and 4, or just on step 4.

Actual result:
- Both views display all issues (that were added on both step 2 and step 4)

Is this ok?
I'd like it to be the "expected result"

**Comment by m19c** [comment]
In my opinion it's the expected behaviour since Issues / PRs are loosely coupled with the view of a project.

**Reply by markvrc** [reply]
I don't mean the issues/PRs. I mean that if I want two different views on the same project - one view with one set of issues, another view with another set of issues. I can't get it with the current functionality.

For example, I want to have a project Game,
and in one view I want to see all the tasks,
and in another view I want to see only tasks related to a new release of the game.
I can't do it with current functionality, because both views will be the same.

**Reply by Venkateshkonda** [reply]
@borseno Yes your correct, I'm also facing same problem on Managing multiple views in same project.
Ideally it should behave two different individual views even it is same project.

Here is the example:

One View having Last Product Release Items.
Another View trying to plan Current Product Release Items. In this case, when we click on New view it is getting all the Previous view items in New views. Ideally both are independent views and it doesn't copy the previous view items in to current new view.

Thanks!

**Comment by rileybroughten** [comment, marked as answer]
Hello! 👋  Providing some documentation on how you can [filter your projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects) to customize and tailor the items that you want to appear in each view 👍

**Reply by insunaa** [reply]
~~Not good enough. If I want to categorize drafts through views I have to do it by giving them a prefix such as "Networking: Fix sockets", but I can't filter for "Networking:" because the ":" makes the filter function interpret it as a field. I can only filter for "Networking", but then I get all drafts/issues that contain the word "Networking", even if it's entirely unrelated to the "Networking:" category.~~

~~You can work around this with Labels *as long as your project is tied to a GitHub repo*. You can't add labels to draft issues in a private project that is *not* tied to a repo~~

Actually: nevermind
